src/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ option(USE_GMP "USE_GMP" ON)
 # development-specific options
 option(CHECK_OLEAN_VERSION "Only load .olean files compiled with the current version of Lean" OFF)
 option(USE_LAKE "Use Lake instead of lean.mk for building core libs from language server" OFF)
+option(COPY_CADICAL "Install a copy of cadical" ON)
```

Comment:
This is not a dev-specifc option, please put it below `INSTALL_LICENSE`. `INSTALL_CADICAL` would likely be a more appropriate name then.